### DISCUSSION TITLE
Is the "OctoPrint calibrate" still needed?

### DISCUSSION THREAD

**Discussion by soonic6** [suchmememanyskill/PrusaSlicer-Ender3-v3-SE-Config#40] [Q&A]
Hello,

with the new update and "[Add M420 S1 (restore bed leveling)](https://github.com/suchmememanyskill/PrusaSlicer-Ender3-v3-SE-Config/issues/35)" is the [OctoPrint calibrate before print](https://github.com/suchmememanyskill/PrusaSlicer-Ender3-v3-SE-Config#octoprint-calibrate-before-print) still recommended an necessary? Or should this be deleted from octoprint?

**Comment by suchmememanyskill** [comment]
These are different features. M420 S1 just loads the last stored mesh. The octoprint calibrate before print well, calibrates the printer before a print. Calibrating before a print will result in a more accurate mesh but takes time

**Comment by soonic6** [comment]
Thanks for your answer, but i have to ask again.
I am new to printing stuff.

So, if i did the "Leveling" from the Printers Display, it stores the mesh in its memory and will load it, when using `M420 S1`?
If i am using then `OctoPrint calibrate before print` befor the print starts, it will do a "leveling" again and will overwrite what was loading with `M420 S1` before?

Bether using one or the other?

**Reply by suchmememanyskill** [reply]
You're completely correct.

It comes down to personal preference really, i like to always re-calibrate so i get a more accurate first layer. Saves me the hassle of failed prints.

**Reply by soonic6** [reply]
okay, thank you for clarification!
